### PR TITLE
Simulated call

### PR DESCRIPTION
**Pull request by VoR0220**
Attempts to run a simulated call whereby no actions affect the chain (my source for this is how they were done previously, but I am more than open to an alternative suggestion)

### CODE REVIEW

File: client/core/core.go [hunk 1]
```diff
@@ -69,50 +69,74 @@ func Send(nodeAddr, signAddr, pubkey, addr, toAddr, amtS, nonceS string) (*txs.S
...
+	}
 
-// 	tx := types.NewCallTxWithNonce(pub, toAddrBytes, dataBytes, amt, gas, fee, int(nonce))
-// 	return tx, nil
-// }
+	tx := txs.NewCallTxWithNonce(pub, toAddrBytes, dataBytes, amt, gas, fee, int(nonce))
+	return tx, nil
+}
 
-// func Name(nodeAddr, signAddr, pubkey, addr, amtS, nonceS, feeS, name, data string) (*txs.NameTx, error) {
```
Comment (benjaminbollen) [inline client/core/core.go]: This module has a set of functions that each construct the correspondingly named transaction format.  Simulated call is already accessible through the RPC, we do not want to confuse that here.

Comment (VoR0220) [inline client/core/core.go]: yeah I messed that up here between all my git checkouts.

Comment (VoR0220) [inline client/core/core.go]: How do I access Simulated Call through the RPC?

**Comment by benjaminbollen** [comment]
will track in #264

**Pull request closed without merging**